x/auth/vesting/types/vesting_account_test.go

```diff
@@ -926,6 +926,83 @@ func TestClawback(t *testing.T) {
 	require.Equal(t, int64(50), stakeAmt.Int64())
 }
 
+func TestClawbackBeforeStartTime(t *testing.T) {
+	c := sdk.NewCoins
```

Comment:
nit: this makes the code harder to read IMO. Just for brevity, let's use `sdk.NewCoins` 🙏 
```suggestion
```